plugins/node/opentelemetry-instrumentation-graphql/src/enum.ts

```diff
@@ -53,3 +53,20 @@ export enum SpanNames {
   SCHEMA_VALIDATE = 'graphql.validateSchema',
   SCHEMA_PARSE = 'graphql.parseSchema',
 }
+
+export const createResolveSpanName = (fieldName: string): string => {
```

Comment:
this file is called enum and these are not enums. maybe rename the file to something more meaningful?

Reply:
Moved to `utils.ts`.